### PR TITLE
Added post about CRIO and alternate runtimes from Runcom.

### PR DESCRIPTION
**Pull request by jberkus**
@runcom please review so that I can post it tommorrow.

### CODE REVIEW

File: source/blog/2017-02-08-crio-runtimes.html.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+[Announced in September as &quot;ocid&quot;](https://www.redhat.com/en/about/blog/running-production-applications-containers-introducing-ocid), CRI-O is a new Kubernetes incubator project which is meant to provide an integration path between OCI conformant runtimes and the kubelet. Specifically, it implements the Container Runtime Interface (CRI) using OCI conformant runtimes.
+CRI-O uses [runc](https://github.com/opencontainers/runc) as its default runtime to run Kubernetes pods.
+
+The good news is you're not locked to runc. CRI-O supports any container runtime as long as it is conformant to the OCI runtime specification. Anyone can plug their favourite runtime according to their needs with little tweaks in CRI-O.
+
+We worked with Intel engineers to give their OCI container runtime, [Clear Containers](https://clearlinux.org/features/intel%C2%AE-clear-containers), the ability to work seamlessly with CRI-O. [In their words](https://github.com/kubernetes-incubator/cri-o/issues/332#issuecomment-275256700):
+
+"It's been very easy to get the right changes merged into [CRI-O], this is a truly open project.""
```
Comment (runcom) [inline source/blog/2017-02-08-crio-runtimes.html.md]: CRI-O here has square brackets which aren't needed

File: source/blog/2017-02-08-crio-runtimes.html.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+author: runcom
+date: 2017-02-08 12:00:00 UTC
+published: true
+comments: true
+tags: kubernetes, cri-o, containers, fedora, atomic
+---
+
+The introduction of the [Container Runtime Interface](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/api/v1alpha1/runtime/api.proto) in Kubernetes 1.5.0 as an alpha feature opens the door for plugging alternative [container runtimes](http://programmableinfrastructure.com/components/container-runtime/) in the [kubelet](https://kubernetes.io/docs/admin/kubelet/) more easily, instead of relying on the default docker runtime. Those new runtimes may include virtual machines based ones, such as runv and Clear Containers, or standard Linux containers runtimes like rkt.
+
+READMORE
```
Comment (runcom) [inline source/blog/2017-02-08-crio-runtimes.html.md]: should this be here?

Comment (jberkus) [inline source/blog/2017-02-08-crio-runtimes.html.md]: READMORE?  yes, that creates a "Read more" break.

File: source/blog/2017-02-08-crio-runtimes.html.md (hunk 1 above)
Comment (jberkus) [inline source/blog/2017-02-08-crio-runtimes.html.md]: those are to indicate that they actually said "ocid", and we're substituting.

**Comment by runcom** [comment]
lgtm

**Pull request merged**